### PR TITLE
Added linters, index.html, style.css , readme.md

### PR DESCRIPTION
**Pull request by stevenmukama**
Added linters
Added Html files
Styled the headings
Updated readme file
Make sure to ignore unrelated files

Thank you for reviewing my PR.

### CODE REVIEW

**Review by stevenmukama — COMMENTED** [review]
I added the social media links to the Readme

File: README.md [hunk 1]
```diff
@@ -1,56 +1,31 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
 
+# Hello Microverse
 > Description the project.
```
Comment (Bondok6) [inline README.md]: - [ ] Your README is well documented 💯, but I think it would be better to add a description to your project 👌

File: README.md [hunk 2]
```diff
@@ -1,56 +1,31 @@
...
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+- html
+- css
 
 
-To get a local copy up and running follow these simple example steps.
+## Live Demo (Not available)
```
Comment (Bondok6) [inline README.md]: - [ ] I suggest removing the live demo section 💯

> We should always be sure our repositories are documented professionally.

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi @stevenmukama 🙋‍♂️, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No Linter errors ✅
- Correct GitHub flow ✅
- Your repo is professional ✅
- Well file structure ✅


## Required Changes ♻️
- [ ] _Check the comments under the review._ ⬇

## Optional suggestions
- [OPTIONAL] I noticed that this PR has a conflict, kindly solve it 💡

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by stevenmukama** [comment]
I have tried to solve the git merge conflict but I failed to solve it.

**Comment by stevenmukama** [comment]
I tried to solve this issue but i have failed.

**Review by topeogunleye — APPROVED** [review]
### Hi @stevenmukama, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good description :heavy_check_mark:
 
## [Optional] suggestions:
* [ ] Please kindly fix your git conflicts. check this [article from GitHub](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-using-the-command-line) on how to fix git conflicts.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — COMMENTED** [review]
### Hello @stevenmukama 

Congratulations! 🎉 🎉 🎉  Your project is complete!  

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**